src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) throws CloneNotSupportedException {
+
+        //Test
+        Car zaz = new Car("zaz", "red", 1800, 200, 5);
+        //zaz.setEngine(6, 1800, 2000, "disel", "akpp");
+        Car clonedZaz = (Car) zaz.clone();

```

Comment:
how we can omit this cast?

Reply:
Fixed by changing the return type in clone() method.